use-case-guides/workday-request-time-off/README.md

```diff
@@ -3,129 +3,96 @@ design_pattern_id: 2
 3. What are the required fields for managers to review time off requests (example: comments, business justification). Which are optional?
+    * This affects what questions will be asked to employees by your copilot when they submit a time off request
 4. Can your employees request time off requests only for themselves, or also for someone else?
    * Note: this guide only allows employees to request time off for themselves. See API research below to understand how you can customize API research to allow time off requests for other employees
+5. Do you want employees to be notified when their time off request is approved through the bot?
+    * Note: this guide only describes how employees can request time off. Please build a new plugin to notify employees when their time off request is approved with the [Events API](https://developer.moveworks.com/creator-studio/quickstart/events/)
 
 # For Workday Administrators
 
-This document provides a step-by-step guide designed for Workday Administrators and HR Systems Engineers, detailing the procedures for creating credentials, assigning roles, and configuring workday effectively. It serves as a practical manual to ensure accurate setup and management of technical resources.
+**Level of Access Requested**:
+1. Configure an API Client for [Workday to integrate with Creator Studio](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday), configuring an Integration System User (ISU) and Integration System Security Group (ISSG) in the process. 
+2. Provide access to the `Worker Data` and `Time Off` Business Objects as part of domain security policies. Elaborated in [Domain Permissions](#domain-permissions-required) section
 
-## Configuration Instructions
 
-### Overview
+## Overview of the Configuration
 
 We will be accessing Workday through the read-only WQL API and the read + write AbsenceManagement REST API, through a Workday Connector in Moveworks. This requires you to:
 
-1. Create a Connector in Creator Studio – [link](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday)
-   * Grant the [domain permissions below](#domain-permissions-required) as part of your Integrations Security Systems Group (ISSG)
-2. Test required APIs in Postman
-3. Build use case in Creator Studio
+1. Integrate Workday with Creator Studio – by following [this guide](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday). After following this guide, you will have
+    * An API Client for Workday
+    * An Integration System User (ISU)
+    * An Integration System Security Group (ISSG)
+    * Relevant [domain security policies listed below](#domain-permissions-required) as part of your Integrations Security Systems Group (ISSG)
 
-### Domain Permissions Required
+> ℹ️ **You will be creating at least two ISUs and two ISSGs, one in a staging environment, and one in production.**
```

Comment:
Style guide rule: no references to CI/CD since every org does it differently

Reply:
This issue still not resolved

Reply:
This also visually looks wrong. Maybe try either indenting it or dropping it altogether
![CleanShot 2024-07-28 at 09 21 53@2x](https://github.com/user-attachments/assets/ccde7d77-7c8d-408c-b09b-1ec61d97e176)